CMakeLists.txt

```diff
@@ -84,10 +84,10 @@ endif()
 message("Document root: ${DOC_ROOT}")
 
 #set debug define for source
-if(CMAKE_BUILD_TYPE MATCHES RELEASE)
+if(CMAKE_BUILD_TYPE MATCHES "(RELEASE|Release)")
```

Comment:
The only question I have is whether this branch should maybe be the general default?

Reply:
I think Release is a good default

Reply:
I meant the setting of `DEBUG=OFF` and `NDEBUG=true` as a general default (without specifying that it is a Release build).

Reply:
The rationale here is, that we're explicitly setting `CMAKE_BUILD_TYPE=None` in our packaging infrastructure to not catch release flags (e.g. optimization flags that are incompatible with what the distribution supports).

Reply:
This is also a good default.

Reply:
This sounds good. Release, Debug for myMPD default compile flags and None for packagers.

Reply:
The `None` is implicit (hence no `else()` block is needed in the CFLAGS/LDFLAGS conditional change).

I'll move the `DEBUG=OFF` and `NDEBUG=true` to an `else()` block then! :)

Reply:
A else() with only a comment could describe the intension.

Reply:
Alright, done! :)